src/STAT.md

```diff
@@ -42,34 +42,34 @@ Mode 0  ___000___000___000___000___000___000________________000
-  1  | Vertical blanking                                                | 4560 dots (1087 µs, 10 scanlines)                                  | VRAM, OAM, CGB palettes
+  2  | Searching OAM for OBJs whose Y coordinate overlap this line | 80 dots (19 µs)                                                    | VRAM, CGB palettes
+  3  | Reading OAM and VRAM to generate the picture                | 168 to 291 dots (40 to 60 µs) depending on sprite count            | None
+  0  | Nothing (horizontal blanking)                               | 85 to 208 dots (20 to 49 µs) depending on previous mode 3 duration | VRAM, OAM, CGB palettes
+  1  | Nothing (vertical blanking)                                 | 4560 dots (1087 µs, 10 scanlines)                                  | VRAM, OAM, CGB palettes
 
 ## Properties of STAT modes
 
 Unlike most game consoles, the Game Boy can pause the dot clock briefly,
-adding dots to mode 3's duration. It routinely takes a 6 to 11 dot
-break to fetch sprite patterns between background tile pattern fetches.
+making Mode 3 longer and Mode 0 shorter. It routinely takes a 6 to 11 dot
+break to fetch an OBJ's tile between background tile pattern fetches.
 On DMG and GBC in DMG mode, mid-scanline writes to [`BGP`](<#FF47 - BGP (BG Palette Data) (R/W) - Non CGB Mode Only>)
-allow observing this behavior, as a sprite delay shifts the effect of a
-write to the left by that many dots.
+allow observing this behavior, as the delay from drawing an OBJ shifts the
+write's effect to the left by that many dots.
 
 Three things are known to pause the dot clock:
 
-- Background scrolling: If `SCX mod 8` is not zero at the start of the scanline, rendering is paused for that many dots while the shifter discards that many pixels from the leftmost tile.
+- Background scrolling: If `SCX % 8` is not zero at the start of the scanline, rendering is paused for that many dots while the shifter discards that many pixels from the leftmost tile.
 - Window: An active window pauses for at least 6 dots, as the background fetching mechanism starts over at the left side of the window.
-- Sprites: Each sprite usually pauses for `11 - min(5, (x + SCX) mod 8)` dots. Because sprite fetch waits for background fetch to finish, a sprite's cost depends on its position relative to the left side of the background tile under it. It's greater if a sprite is directly aligned over the background tile, less if the sprite is to the right. If the sprite's left side is over the window, use `255 - WX` for `SCX` in this formula.
+- Sprites: Each sprite usually pauses for `11 - min(5, (x + SCX) % 8)` dots. Because sprite fetch waits for background fetch to finish, a sprite's cost depends on its position relative to the left side of the background tile under it. It's greater if a sprite is directly aligned over the background tile, less if the sprite is to the right. If the sprite's left side is over the window, use `255 - WX` for `SCX` in this formula.
```

Comment:
```suggestion
- Sprites: Each sprite usually pauses for `11 - min(5, (x + SCX) % 8)` dots. Because sprite fetch waits for background fetch to finish, a sprite's cost depends on its position relative to the left side of the background tile under it. It's greater if a sprite is directly aligned over the background tile, less if the sprite is to the right. If the sprite's left side is over the window, use `255 - WX` instead of `SCX` in this formula.
```